### ISSUE TITLE
Add more checks to constructor.

### ISSUE THREAD

**Issue by rashtrakoff** [DAM-Protocol/Stroller-Protocol#12]
Location: StrollManager.sol - Line 27 - 35

Description:
There is no logic in the contract that actually enforces minLower to be less than minUpper and vice versa. As there is no way to change the values of minLower and minUpper, it should be ensured that they are correct. Additionally, if minLower is set to 0, this would allow for unnecessary 0 TopUps by the Keepers to potentially happen.

Justification:
Just wanted to save some gas by avoiding trivial checks. However, a good point has been raised as we need a way to change `minUpper` and `minLower` parameters.

Solution:
Add some trivial checks but more importantly, add functions to change `minLower` and `minUpper` limits. It's worth noting that existing top-ups whose limits may not adhere changed global limits i.e., `minLower` > `userTopUpLowerLimit` or `minUpper` > `userTopUpUpperLimit`; we need to enforce the global limits on them as well.

**Comment by ngmachado** [comment]
@rashtrakoff do you have a intuition about the global limits?

**Comment by rashtrakoff** [comment]
> @rashtrakoff do you have a intuition about the global limits?  

The larger the limits, the safer it is. With your previous experience of running the sentinels, what do you recommend? Anyway, we also plan to add the functions to modify these parameters as it's something we missed adding before the audit. Do you think that's fine?

**Comment by rashtrakoff** [comment]
Agreed, however, should we add a function to increase or decrease the limits or should we do that in the next release? Because, as I said in the note in the issue description, the limits have to be enforced for the existing top ups which can be out of the new limits range.

**Comment by rashtrakoff** [comment]
PR #31 fixes this.
